runtime/testy/Ansible.py

```diff
@@ -251,6 +252,16 @@ def __init__(self, badThingsQueue, stateQueue, pipe):
         self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.sock.connect((self.dawn_ip, TCP_PORT))
 
+        proto_message = notification_pb2.Notification()
+        proto_message.header = notification_pb2.Notification.SENSOR_MAPPING
+        with open('namedPeripherals.csv', 'r') as f:
+            sensorMappings = csv.reader(f)
+            for row in sensorMappings:
+                pair = proto_message.sensor_mapping.add()
+                pair.device_new_name = row[0]
```

Comment:
Uhh so the sensor mapping should relate some hibike-related uids or device ids to human-readable names.

The use of `device_new_name` and `device_name` as the names for the proto fields is highly misleading if this is the case.